packages/@okta/vuepress-site/docs/reference/error-codes/index.md

```diff
@@ -1,13 +1,13 @@
 ---
-title: Error Codes
+title: Examples of Error Codes and Messages
 excerpt: Understand Okta API errors.
 ---
 
-# Overview
+# Examples of Error Codes and Messages
```

Comment:
just realized we call them Error "Descriptions" in the table, should they be that or "Messages"? Either way, lets make it consistent